test/runtests.jl

```diff
@@ -118,9 +123,30 @@ let callsites = find_callsites_by_ftt(f_matches, Tuple{Any, Any}; optimize=false
     @test callinfo isa Cthulhu.MultiCallInfo
 end
 
-@testset "union-split constant-prop'ed callsites" begin
-    anonymous_module() = Core.eval(@__MODULE__, :(module $(gensym()) end))::Module
+# NOTE some inference won't work with inner functions
+anonymous_module() = Core.eval(@__MODULE__, :(module $(gensym()) end))::Module
```

Comment:
Just FYI, I think you can just use `Module()` for this, but doesn't really matter here.

Reply:
That's neat. Thanks.